### ISSUE TITLE
Oven does not show up

### ISSUE THREAD

**Issue by squirtbrnr** [simbaja/ha_gehome#23]
I have a GE Profile convection oven that does not show up under this integration.  I have tried reinstalling the custom component, rebooting HA, and verifying the oven shows up in the SmartHQ app for GE.  There is an error that keeps popping up in my HA log.

```
2021-07-28 05:19:46 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/ge_home/update_coordinator.py", line 287, in on_device_initial_update
    self.maybe_add_appliance_api(appliance)
  File "/config/custom_components/ge_home/update_coordinator.py", line 121, in maybe_add_appliance_api
    api.build_entities_list()
  File "/config/custom_components/ge_home/devices/base.py", line 112, in build_entities_list
    e for e in self.get_all_entities()
  File "/config/custom_components/ge_home/devices/oven.py", line 34, in get_all_entities
    cooktop_config: ErdCooktopConfig = self.appliance.get_erd_value(ErdCode.COOKTOP_CONFIG)
  File "/usr/local/lib/python3.9/site-packages/gehomesdk/ge_appliance.py", line 119, in get_erd_value
    return self._property_cache[erd_code]
KeyError: <ErdCode.COOKTOP_CONFIG: '0x551c'>
```

**Comment by simbaja** [comment]
Yep, this issue was found and should be fixed in the "dev" branch.  Can you clone that branch and place into custom components and confirm if it resolves your issue?

**Comment by simbaja** [comment]
Should be fixed in 0.4.0, feel free to re-open if you're still running into an issue.
